config/locales/views/components/observation.yml

```diff
@@ -8,9 +8,9 @@ en:
           other: <a href="%{school_path}">%{school}</a> scored <strong>%{count} points</strong> after they recorded "<a href="%{compact_path}">%{target}</a>"
           zero: <a href="%{school_path}">%{school}</a> recorded "<a href="%{compact_path}">%{target}</a>"
         description_html:
-          one: Recorded "<a href="%{compact_path}">%{target}</a>"
-          other: Recorded "<a href="%{compact_path}">%{target}</a>"
-          zero: Recorded "<a href="%{compact_path}">%{target}</a>"
+          one: <a href="%{compact_path}">%{target}</a>
```

Comment:
These probably don't need to be in translations any more as there's no text left? but could leave for now in case we change our minds?

Reply:
Yeah, that's what I was thinking